pos_customer_wallet/static/src/xml/pos.xml

```diff
@@ -24,4 +24,45 @@
             </div>
         </t>
     </t>
+
+    <t t-extend="PosTicket">
+        <!--
+            Customer must see his balance even if it does not use
+            customer wallet as a payement method for this order.
+            Because it's the only way he can check his balance.
+            Also this balance should not been shown if the user is not
+            allowed to use customer wallet as a payment method.
```

Comment:
I think the word 'allowed' is misleading here. There's no logic (yet?) forbidding certain partners from using the customer wallet.